string_assignment.js

```diff
@@ -0,0 +1,133 @@
+
+// String Exercises
+// 1.
+// console.log the first character of a string.
+// let str = 'Good'
+// console.log(str[0])
+// ------------------------------------------------
+// 2.
+// console.log the length of a string.
+// console.log(str.length)
+//-------------------------------------------------
+// 3.
+// console.log the last character of any string.
+// let word = 'Good morning!'
+// console.log(word[11])
+// console.log(word[word.length-1])
+// -----------------------------------------------------------------------------------------------------------------------------------------
+// 4.
+// Create a code block that takes a single string variable, and returns a copy of the string with the last letter capitalized. For example:
+
+// 'jimmy'
+// => 'jimmY'
+// let lastName = 'bazi'
+// // lastName = lastName[0] + lastName[1] + lastName[2] + lastName[3].toUpperCase()
+// // console.log(lastName)
+// let capitalizedLast = lastName.slice(-1).toUpperCase()
+// let firstLetters = lastName.slice(0, lastName.length - 1)
+// console.log(capitalizedLast)
+// console.log(firstLetters)
+// console.log(firstLetters + capitalizedLast)
+
+// ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+// 5.
+// Create a drEvil code block that will take a single number variable, and log the ' dollars', your code block should add '(pinky)' at the end of the amount if it's 1 million. For example:
+// let amount = 10
+// // => 10 dollars
+// let amount = 1000000
+// // =>  1000000 dollars (pinky)
+// ------------------------------
+//
+// let word = "dollars"
+// let num = 20
+// let sentence = num + ' ' + word
+// console.log(sentence)
+// let x = 20
+// let sentence = `${x} dollars`
+// console.log(sentence)
+// --------------------------------
+// let x = 2000000
+// let sentence = `${x} dollars \(pinky)`
+// console.log(sentence)
+// ---------------------------------------------------------------------------------------------------------------------
+// 6.
+// Create a verbing code block. It should take a single string variable. If its length is at least 3, it should add 'ing' to its end, unless it already ends in 'ing', in which case it should add 'ly' instead. If the string length is less than 3, it should leave it unchanged. For example:
+// verbing('box')
+// // => 'boxing'
+// verbing('train')
+// // => 'training'
+// verbing('swimming')
+// // =>  'swimmingly'
+// verbing('go')
+// =>  'go'
+// ----------------------------------------------
+
+// let verb = 'train'
+//  1st case verb already ends in "ing"
+// ten append 'ly' at the end
+//
+// 2nd case verb.leghth > 3. not end in already 'ing'
+//  append 'ing' at the ends
+//  3rd case.length < 3. console
+//==>'walking'
+// if(verb.endsWith('ing')){
```

Comment:
Good use of string method "endsWith" here. Would there be other way to solve this problem? Perhaps if statement that says (str.slice(-3) === "ing")?